README.md

```diff
@@ -1,2 +1,183 @@
 # AndcultureCode.CSharp.Sitefinity.Testing
```

Comment:
Praise: Thank you very much for keeping the documentation up to date as you continue to build things out. It is often a step overlooked, so I am happy to see what you’ve put together here.